### ISSUE TITLE
Nice job removing the client_id for security, but I am unable to test your site ^_^ 

### ISSUE THREAD

**Issue by jzhu01** [mrrichardasmith/Jamming#1]
Seen here: https://github.com/mrrichardasmith/Jamming/blob/4d79727a385e093249e1554182b265245de47b04/src/util/Spotify.js#L2

Great job removing the client_id for security, but this poses an issue for me since I am unable to test your site! :)

**Comment by mrrichardasmith** [comment]
Do you need me to send my CLIENT_ID I read in the slack channel that we should not be submitting the CLIENT_ID in the our git pushes.

**Comment by jzhu01** [comment]
Nope, it's all good! I wasn't able to load your project, but reviewing the code gave me a pretty good sense of what it does. :)

**Comment by mp1pro** [comment]
@jzhu01 In this case you would use your client id to test the app.

**Comment by jzhu01** [comment]
@mp1pro thanks for the heads up!
